java/code/src/com/redhat/rhn/domain/channel/Channel.hbm.xml

```diff
@@ -145,6 +145,42 @@ PUBLIC "-//Hibernate/Hibernate Mapping DTD 3.0//EN"
         -->
     </class>
 
+    <sql-query name="Channel.findAllByUserOrderByChild">
+        <![CDATA[
+    SELECT {c.*}
+    FROM rhnChannel {c}
+        LEFT OUTER JOIN rhnChannel parent ON c.parent_channel = parent.id
+        LEFT OUTER JOIN rhnChannelCloned c_1_ ON c.id = c_1_.id
```

Comment:
Do we need to join with rhnChannelCloned? I think it is not returned either used in any place on the query.

Reply:
Without that, hibernate throws an error.